test/get.sh

```diff
@@ -646,6 +646,10 @@ it_can_get_returned_ref() {
   test "$(cat $dest/.git/ref)" = "${ref3}" || \
     ( echo ".git/ref does not match. Expected '${ref3}', got '$(cat $dest/.git/ref)'"; return 1 )
 
+  test -e $dest/.git/tag_ref || ( echo ".git/tag_ref does not exist."; return 1 )
+  test "$(cat $dest/.git/tag_ref)" = "${ref3}" || \
```

Comment:
Unless I'm missing something, this doesn't seem to test that we actually output `git describe` as it ends up just being a regular ol' commit. Could we add a test that sets up tags and such and asserts against the specific tag name?